aml-development/frontend
# task(access): limit user access via privilage

Adds a function to check user's access rights.
Implemented into listing page.

Closes #AMLOS-707

To Test:
Pull this branch.
open as 4 users:
Bigbrother, wsmith, jones, and johnson

("access to" means can edit, and access to the admin section)
Bigbrother should have access to everything,
Wsmith should have access to listings part of minitrue
Jones should have access to their own listings
and johnson should have access to none.
